apps/verticalhome/js/pin_app_navigation.js

```diff
@@ -7,37 +7,61 @@
    * This is PinAppNavigation's constructor.
    * @param {Object} list [DOM container that contains pin elements]
    */
-  function PinAppNavigation(list) {
-    if (!list && typeof list !== 'object') {
-      console.error('List of pin apps must be an object');
-      return;
+  function PinAppNavigation(list, selector) {
+    if (!list) {
+      console.error('List must be a dom container with navigated elements');
     }
 
-    this.elemList = list;
-    window.removeEventListener('keydown', this);
-    window.addEventListener('keydown', this);
-  }
+    /**
+     * This variable contains all elements that will get using this.selector
```

Comment:
If I understand the comment correctly:
 elements that will get using -> elements which will be obtained